.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!-- Thanks for contributing to CorsixTH's Community Content! -->
+<!-- Please fill out the form below, and find guidance inside these comment tags -->
+<!-- Full guidance is available at https://github.com/CorsixTH/CorsixTH/wiki/Submitting-to-Community-Content -->
+<!-- Checkbox items should be presented as [ ] for no/false; and [x] for yes/true -->
+
+# Community Content Contribution
+- [ ] I am submitting a standalone level **OR**
+- [ ] I am submitting a campaign
+
+**Name of level/campaign <!-- delete as appropriate -->:**
+**(If relevant) Levels of campaign:**
+
+## Campaign Data
+- [ ] Not applicable
+- [ ] I have included a `.campaign` file with this submission
+
+## Game Data
+<!-- Please copy and paste the "Level Name", .level, and .map sections below for each level in the submission if you are submitting a campaign -->
+<!-- Modified someone else's work? Please add this to the declarations -->
+**Level Name:**
+The `.level` file is:
+- [ ] From CorsixTH (specify level file name)
+- [ ] From a Community Content submission (specify level file name and original author)
+- [ ] My own creation and is included in this submission
+
+*Level Declarations (as guided above):*
+
+The `.map` file is appropriately pointed to by the level file and is:
+- [ ] From CorsixTH (specify level file name)
+- [ ] From a Community Content submission (specify level file and original author)
+- [ ] My own creation and is included in this submission
+
+*Map Declarations (as guided above):*
+
+## General Declaration
+<!-- Please read this before accepting the below -->
+*By submitting this pull request I confirm that I am the author of this work. Where portions of this work have been created by other authors (including modifications) I have appropriately declared their authorship.*
```

Comment:
We may need to be more explicit that the original author has checked the `other works` have appropriate licensing to allow sharing/modification

Reply:
can you ask a yes/no question?

- [ ] I checked that all files not made by me are allowed to be bundled with this submission.
- [ ] I am the author.

"please read" doesn't work (or at least we cannot verify it), ask specifically for confirmation, if possible.

Reply:
Yes we can probably be more direct with the statements, was in my mind at the time but some reason didn't do it.